kubectl.ts

```diff
@@ -28,11 +28,9 @@ async function run() {
         tl.debug("DEBUG:  " + kubectlbinary + " apply -f " + yamlfile + " --kubeconfig ./config")
 
         let kubectl: ToolRunner = tl.tool(kubectlbinary).arg('apply').arg('-f').arg(yamlfile).arg('--kubeconfig').arg('./config');
-        let result =  kubectl.execSync();
-        tl.debug("STDOUT: " + result.stdout);
-        tl.debug("STDERR: " + result.stdout);
+        await kubectl.exec();
 
-        tl.setResult(tl.TaskResult.Succeeded, "kubectl works.")
+        tl.setResult(tl.TaskResult.Succeeded, "kubectl works.");
```

Comment:
this semicolon wasn't necessary, i just happened to notice and adding it appeared to be consistent with the style present in the file :)